string mode_command
---
int32 result